PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/networking/parameters/BasicParameterBuilder.java

```diff
@@ -209,7 +210,12 @@ private void setVideoImpValues(Imp imp) {
 
                 video.minbitrate = videoParameters.getMinBitrate();
                 video.maxbitrate = videoParameters.getMaxBitrate();
-
+                video.linearity = videoParameters.getLinearity();
+                if (videoParameters.getPlacement() != null) {
+                    video.placement = videoParameters.getPlacement().getValue();
+                } else if (adConfiguration.getPlacementTypeValue() != PlacementType.IN_BANNER.getValue()){
+                    video.placement = PlacementType.INTERSTITIAL.getValue();
```

Comment:
It looks incorrect. We should set `placement` based on the ad unit type or user-defined property. If user set IN-FEED the SDK should send it.